src/main/java/de/tum/in/www1/exerciseapp/service/BambooService.java

```diff
@@ -580,6 +580,10 @@ private ResponseEntity retrievArtifactPage(String url) throws BambooException {
      * - isBuilding: true if the plan is building
      */
     public Map<String, Boolean> retrieveBuildStatus(String planKey) {
+        if(planKey.toLowerCase().equals("tutorial")){
```

Comment:
this should fix the nullpointer excpetion

Reply:
If you do `"tutorial".equals(planKey.toLowerCase()` you will also prevent the NullPointerException because `.equals(...)` is never called on `null`.

Reply:
nice idea changed it